lib/please_cli/please_cli/config.py

```diff
@@ -406,6 +406,57 @@
                     },
                 },
             },
+            {
+                'target': 'HEROKU',
+                'options': {
+                    'testing': {
+                        'nix_path_attribute': 'docker',
+                        'heroku_app': 'releng-testing-tooltool',
+                        'heroku_dyno_type': 'worker',
+                        'heroku_command': 'flask worker',
+                    },
+                    'staging': {
+                        'nix_path_attribute': 'docker',
+                        'heroku_app': 'releng-staging-tooltool',
+                        'heroku_dyno_type': 'worker',
+                        'heroku_command': 'flask worker',
```

Comment:
@La0 you might be interested to know that you can with this PR you can specify `heroku_command` for any `HEROKU` deployment target which lets you reuse docker image.